sdcm/nemesis.py

```diff
@@ -2055,7 +2055,10 @@ def _get_all_tables_with_no_compact_storage(self, tables_to_skip=None):
                     to_be_skipped = []
                 else:
                     to_be_skipped = to_be_skipped.split(',') + to_be_skipped_default
-                tables = get_db_tables(session, ks, with_compact_storage=False)
+                tables = get_db_tables(session=session,
+                                       keyspace_name=ks,
+                                       node=self.tester.db_cluster.nodes[0],
```

Comment:
Are we sure that this first node is always alive considering that it may be disrupted by concurrent nemesis?

Reply:
Ok, this node is used for session create. But I think you are right. Changing it to target_node